### DISCUSSION TITLE
Ambuiguita R012

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#868] [Supporto servizi cooperativi] [answered]
Salve, potete spiegarmi la differenza di "stato presa visione"  nei casi di passaggio del solo IdPresaVisione e nel caso di IdEvento e idPresaVisione: 
![image](https://github.com/italia/ansc/assets/112756480/8f6eb6f3-d5a8-442d-906e-a69e3c51252b)

**Comment by mttfranci** [comment, marked as answer]
Salve se viene passato l'idPresaVisione, viene restituito lo stato della firma dello specifico firmatario (visto anche che potrebbero esserci più firmatari che hanno optato per la firma elettrocnia per lo stesso evento). Se invece viene indicato il solo idEvento, lo stato è quello complessivo delle firme dell'evento stesso.

Saluti

Assistenza ANSC
